README.md

```
@@ -29,6 +29,56 @@ packages reappear if they are still installed. Review `git diff Brewfile`
+
+Skills are invoked with plugin namespaces (e.g. `/git-workflow:commit`,
+`/dev-workflow:spec`). Environment-coupled skills (cmux, herdr, iterm2,
+hunk-review, etc.) remain in `.claude/skills/` here.
+
+### Setup on a new machine
+
+Both marketplaces are private repos, so SSH access to GitHub (or `gh auth
+login`) must work first. Then:
+
+```shell
+$ cd $HOME/dotfiles && git pull   # syncs .claude/settings.json via symlink
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)